doc/quick-start/qs-guide.rst

```diff
@@ -7,82 +7,119 @@ This guide will help you to set up thing's development environment, then build a
 
 Thing's Development Environment
 -------------------------------
+In order to compile and flash applications for the KNoT Thing, it's necessary to set up the development environment.
 
-The fastest way to use or to develop for KNoT thing is to use the `KNoT Thing Docker container <../thing/thing-docker.html>`_.
+The fastest way to do it is by using a pre-built Docker image.
 
-Another way is to get the KNoT Thing source code by following all steps listed on `requirements <../thing/thing-requirements.html>`_.
+1. Download and install `Docker <https://docs.docker.com/install/>`_.
 
-After this set up, it will be possible to use the `KNoT CLI application <../thing/thing.cli.html>`_.
+#. In a terminal, get the latest Docker environment image.
```

Comment:
Wouldn't it be better to use KNoT Thing SDK Docker environment image instead of Docker environment image?